template/page-events.html.jinja

```diff
@@ -430,10 +434,37 @@
     "image": "Hackathons.png",
     "end_date": "August 31, 2023"
     },
+    { "name": "XRPL South Korea Meetup - XCCESS",      
+    "description": "We are excited to introduce the XRP Ledger XCCESS - an exclusive meetup bringing together the brightest minds, innovators, and enthusiasts from the South Korea Blockchain Industry. Join us for an engaging experience during the Korea Blockchain Week.",
+    "type": "meetup",
+    "link": "https://lu.ma/xrplxccess",
+    "location": "South Korea - JBK Tower",
+    "date": "September 06, 2023",
+    "image": "SouthKoreaMeetup.png",
+    "end_date": "September 06, 2023"
+    },
+    { "name": "XRPL Grants Info Session: Financial Inclusion Focused",      
```

Comment:
Should these be ordered by date? So, events on Sept 06 are listed before Sept 05.